doc/README.md

```diff
@@ -15,27 +24,18 @@ Or, if you'd rather not build the example gallery:
 
     make html-noplot
 
-That should create all the doc in directory _build/html
+That should create all the doc in directory `_build/html`
 
 To build the PDF manual, run
 
     make latexpdf
 
-The website is hosted at github and can be updated manually (for releases)
-by pushing to the https://github.com/scikit-learn/scikit-learn.github.io repository.
-
-It's recommended to run OptiPNG, before uploading the website.
-The PNG files generated by Matplotlib tend to be ~20% too big, and they're
-costing us bandwidth. You can run OptiPNG with::
-
-    make optipng
-
-# Development documentation automated build
+Make sure you first have the correct version of scikit-learn
 
-A Rackspace cloud server named 'docbuilder' is continuously building the master branch
-to update the http://scikit-learn.org/dev tree of the website.
+## Hosting and automatic builds
 
-The configuration of this server is managed at:
-
-  https://github.com/scikit-learn/sklearn-docbuilder
+The website is hosted at github, but should rarely be updated manually
+by pushing to the https://github.com/scikit-learn/scikit-learn.github.io repository.
 
+Most updates can be made by pushing to master (for /dev) or a release branch
+like 0.00.X, from which Circle CI builds and uploads documentation.
```

Comment:
Maybe  `0.19.X` would be a better example of a release branch than `0.00.X`? 

Also, could it be worth linking to [this section of the docs](http://scikit-learn.org/dev/developers/tips.html#viewing-the-rendered-html-documentation-for-a-pull-request)  somewhere in this paragraph?